sklearn/utils/_mocking.py

```diff
@@ -61,6 +61,12 @@ class CheckingClassifier(ClassifierMixin, BaseEstimator):
     check_X
     foo_param
     expected_fit_params
+
+    Attributes
+    ----------
+    classes_ : array, shape = (n_classes,)
```

Comment:
That is the only thing I am not sure about ...

Should we "force" the documentation on some classes that are only used for tests like `CheckingClassifier`? I would be enclined to skip the test instead but I don't have a strong opinion.

An alternative is to do a minimal documentation as was done for the parameters a few lines above, i.e.:
```
Attributes
----------
classes_
```

Reply:
ok. Definitely more consistent with the parameter documentation. Doing it right now.
don't know about skipping the test.

Reply:
The minimal documentation approach is simpler, so let's do this and see whether others have a different opinion.